src/service-worker.js

```diff
@@ -1,6 +1,74 @@
 /* eslint-env serviceworker */
+import doRequest from './do-request';
+import DeviceStateStore from './device-state-store';
+
 self.PusherPushNotifications = {
+  endpointOverride: null,
   onNotificationReceived: null,
+
+  _endpoint: instanceId =>
+    self.PusherPushNotifications.endpointOverride
+      ? self.PusherPushNotifications.endpointOverride
```

Comment:
How do developers set this? Should it be passed from the client?

Reply:
Set after `importScripts` - Not intended for public use though